public/js/components/Scopes.js

```diff
@@ -39,7 +39,7 @@ function getSpecialVariables(pauseInfo, parentName) {
     vars.push({
       name: "<exception>",
       path: parentName + "/<exception>",
-      contents: { value: thrown.toJS() }
+      contents: { value: thrown }
```

Comment:
This change does not work for me. I don't know what edge case you hit, but please look into it and figure out what happened.

This is what it looks like without `toJS`:

<img width="295" alt="screen shot 2016-08-12 at 11 16 24 am" src="https://cloud.githubusercontent.com/assets/17031/17627176/582f8432-607e-11e6-9d9e-3031c008b7b4.png">

If I call `toJS`, it shows me all the right fields:

<img width="305" alt="screen shot 2016-08-12 at 11 15 45 am" src="https://cloud.githubusercontent.com/assets/17031/17627193/5fd7aff2-607e-11e6-84e7-1e78c60c3c72.png">